### PR TITLE
Landing

### PR DESCRIPTION
**Pull request by abolayo**
i added my landing page
i added herbals.css file
i added two image files

### CODE REVIEW

**Comment by kenny-wealth** [comment]
Please give the PR a more descriptive title

File: submissions/medinatapanpa/homepage2/index.html [hunk 1]
```diff
@@ -0,0 +1,158 @@
+<!doctype html>
+<html>
+    <head>
+        <title> Herbal</title>
+        <link rel="stylesheet" href="herbal.css">
+       
+        <header>
+                    <div class="topnav">
+                        <img class="logo"src="pistachios.jpg" alt="herbs" class= "pics">
```
Comment (kenny-wealth) [inline submissions/medinatapanpa/homepage2/index.html:9]: `pistachios.jpg` was not checked in

File: submissions/medinatapanpa/homepage2/index.html [hunk 2]
```diff
@@ -0,0 +1,158 @@
...
+                                <h3>Tyme</h3>
+                                <p class="price"> From 199.99</p>
+                                <p>Be it herbs from the Mediterranean, from China, Japan even Arabian ...</p>
+                                <p><button>Add to Cart</button></p>
+                            </div>
+                  
+            </div>
+        
+    </body>
+    <footer>
```
Comment (kenny-wealth) [inline submissions/medinatapanpa/homepage2/index.html:138]: Footer is part of the body section

File: submissions/medinatapanpa/homepage2/index.html [hunk 3]
```diff
@@ -0,0 +1,158 @@
+<!doctype html>
```
Comment (kenny-wealth) [inline submissions/medinatapanpa/homepage2/index.html:1]: Generally review indentations using the Styleguide provided

File: submissions/medinatapanpa/homepage2/herbal.css [hunk 4]
```diff
@@ -0,0 +1,98 @@
+/* Add a black background color to the top navigation 
+.topnav {
```
Comment (kenny-wealth) [inline submissions/medinatapanpa/homepage2/herbal.css:2]: Please remove if it's not needed

**Review by kenny-wealth — CHANGES_REQUESTED** [review]

File: submissions/medinatapanpa/homepage2/index.html (hunk 1 above)
Comment (abolayo) [reply submissions/medinatapanpa/homepage2/index.html:9]: Resolved

File: submissions/medinatapanpa/homepage2/index.html (hunk 2 above)
Comment (abolayo) [reply submissions/medinatapanpa/homepage2/index.html:138]: Noted

File: submissions/medinatapanpa/homepage2/herbal.css (hunk 4 above)
Comment (abolayo) [reply submissions/medinatapanpa/homepage2/herbal.css:2]: Done

**Comment by kunmi02** [comment]
> i added my landing page
> i added herbals.css file
> i added two image files

**Pull request closed without merging**